6_Model Comparison.ipynb

```diff
@@ -0,0 +1,6627 @@
+{
```

Comment:
<p>Add a semicolon at the end of the sigma definition so it doesn't print out the array you defined.</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.